src/components/BannerCarousel/style.scss

```diff
@@ -0,0 +1,38 @@
+.carousel-container {
+  box-shadow: 0 8px 6px -6px var(--box-shadow);
+  margin: 0 12%;
+  button {
+    background-color: grey;
+    color: white;
+    height: 30;
+    opacity: 0.5;
+    position: absolute;
+    top: calc(50% - 15px);
+    width: auto;
+    z-index: 2;
+  }
+  .custom-control-dots {
+    background: whitesmoke;
+    border-radius: 50%;
+    display: inline-block;
+    list-style-type: circle;
+    margin: 0 8px;
+    height: 8px;
+    width: 8px;
+    &.selected {
+      background: grey;
+    }
+  }
+}
+
+@media (max-width: 480px) {
+  .carousel-container {
+    margin: 0;
+  }
+}
+
+@media (min-width: 481px) and (max-width: 768px) {
+  .carousel-container {
+    margin: 0;
+  }
+}
```

Comment:
dont directly use color names in the css. Create a separate file and put all colors in it as variables, then use those variables

Reply:
Global CSS Variables added.